### PR TITLE
Enable identitiy verification user hash by default

### PR DESCRIPTION
**Pull request by kmiyashiro**
## Context

We now are able to send pings with `user_hash` without breaking anything, even if Identity Verification is not enabled for an Intercom app's settings. That means there is no downside to always enabling Identity Verification on the Messenger side.

## Problem

Identity verification should be enabled by everyone, we should make it as easy as possible to enable.

## Approach

Enable identity verification by default in the plugin. We already have the secret, so we can just always generate the hash if we can. This removes the step to manually enable it in the plugin settings, the only step left is to enable it in the Intercom app settings.

screen shot 2017-10-25 at 10 24 05 am

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -423,6 +396,12 @@ private function validateRawData($raw_data)
   }
 }
 
+if (getenv('INTERCOM_PLUGIN_TEST') == '1' && !function_exists('apply_filters')) {
```
Comment (jeobrien) [inline bootstrap.php]: What is `INTERCOM_PLUGIN_TEST`?

Comment (mckenna) [reply bootstrap.php]: It's set in `.travis.yml` — I'm guessing it's a variable to check if the plugin is currently in test mode?

Comment (kmiyashiro) [reply bootstrap.php]: Yup

**Comment by apassant** [comment]
@kmiyashiro can you please update the version number: https://github.com/intercom/intercom-wordpress/blob/master/bootstrap.php#L8

I'll merge on Monday and ship the update to https://wordpress.org/plugins/intercom/

**Comment by kmiyashiro** [comment]
@apassant done!

**Pull request merged**